quantum/quantum_keycodes.h

```diff
@@ -462,6 +462,38 @@ enum quantum_keycodes {
     UNICODE_MODE_BSD,
     UNICODE_MODE_WINC,
 
+#ifdef EXTRAKEY_ENABLE
```

Comment:
I don't think it makes sense to add `#ifdef EXTRAKEY_ENABLE` in `quantum_keycodes.h`, I think if there is going to be a reorganization of keycodes, then this is a good opportunity to actually remove all the `#ifdef` in `quantum_keycodes.h` so that enum integer values are constant, regardless of features being enabled. I know this would mean it's possible for firmware to be built with keymaps containing keycodes that "do nothing" if the associated feature is disabled, but that happens now anyway (e.g. mouse keys, extra keys), and would also allow community layout keymaps to still compile if the keyboard implementation has disable features.

Reply:
I though it was related to the resulting firmware size but oddly enough it seems that it's not the case. I pushed another commit where I removed the ifdef there and moved others in another couple of places to make it compile whether EXTRAKEY_ENABLE is defined or not.